services/src/main/java/org/exoplatform/webconferencing/WebConferencingService.java

```diff
@@ -3930,5 +3938,30 @@ private void broacastCallEvent(String eventName, CallInfo call, String userId, D
     this.broacastCallEvent(eventName, call, userId, fileSize, STATUS_OK);
   }
 
+  private void broacastCallEvent(String eventName, CallInfo call, String userId, String fileName, Node node,UploadFileInfo uploadFileInfo, String status) {
+    try {
+      Map<String, String> info = new HashMap<>();
+      info.put("user_id", userId);
+      info.put("status", status);
+      DocumentService documentService = WCMCoreUtils.getService(DocumentService.class);
+      if (node != null) {
+        String fileUrl=null;
+        info.put("file_name", fileName);
+        if(uploadFileInfo.getType().equals(OWNER_TYPE_SPACE)) {
+          Space space = spaceService.getSpaceByPrettyName(uploadFileInfo.getIdentity());
+          String groupId = space.getGroupId();
+          fileUrl = documentService.getDocumentUrlInSpaceDocuments(node, groupId);
+        } else {
+          fileUrl = documentService.getDocumentUrlInPersonalDocuments( node, userId) + "?optional=true";
```

Comment:
will this return url of new documents app ?

Reply:
we need a better solution because the documents app wil replace the old one within a few weeks
@mkrout @rdenarie please sync